source/partials/next_release/_release.md.erb

```diff
@@ -1,23 +1,48 @@
-<h4>Some feature</h4>
+<h4>Introduction of new attributes</h4>
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Introduced `username` and `password` (or `encryptedPassword`) attributes on Git and Mercurial materials. Users can now use these fields to setup credentials in a secure manner, instead of putting credentials in the URL. The YAML, JSON and Groovy config repo plugins have been updated to allow these additional fields on material definitions.
```

Comment:
`The YAML, JSON and Groovy config repo plugins` =>  please add a link to the plugin?